3p/iframe-messaging-client.js

```diff
@@ -103,11 +103,19 @@ export class IframeMessagingClient {
     });
   }
 
+  /**
+   * @param {!Window} win
+   */
   setHostWindow(win) {
+    /** @private {!Window} */
```

Comment:
I'm wondering if this is for code style or it makes a difference to the compiler?